models/public/ctdet_coco_dlav0_384-onnx/ctdet_coco_dlav0_384-onnx.md

```diff
@@ -0,0 +1,130 @@
+# ctdet_coco_dlav0_384-onnx
+
+## Use Case and High-Level Description
+
+CenterNet object detection model `ctdet_coco_dlav0_384-onnx` originally trained on PyTorch\*
+than converted to ONNX\* format. CenterNet model an object as a single point - the center point of its bounding box
+and uses keypoint estimation to find center points and regresses to object size.
+For details see [paper](http://arxiv.org/abs/1904.07850), [repository](https://github.com/xingyizhou/CenterNet/).
+
+### Steps to Reproduce PyTorch to ONNX Conversion
+By default model is provided in ONNX format, which was obtained by the following steps.
+
+1. Clone the original repository
+```sh
+git clone https://github.com/xingyizhou/CenterNet
+cd CenterNet
+```
+2. (Optional) Checkout the commit that the conversion was tested on:
+```sh
+git checkout 8ef87b4
+```
+3. Modify the `src/lib/models/networks/dlav0.py` file, lines 605-607 to
+```python
+ret = []
+for head in sorted(self.heads.keys()):
+   ret.append(self.__getattr__(head)(x))
+```
```

Comment:
I think it would be better to save a patch file in the model directory and refer to that. A patch file is easier to apply automatically and to retarget to a different version of the original code.